courses/free-courses-id.md

```diff
@@ -219,7 +219,8 @@
 * [ReactJS Untuk Pemula](https://www.petanikode.com/reactjs-untuk-pemula/) - Petani Kode
 * [Tutorial React JS Bahasa Indonesia](https://www.youtube.com/playlist?list=PLp6BJq2fT_g91yCNCWi_bIe-ng7S7rt6V) - Lampung JS
 * [Tutorial React Native Bahasa Indonesia (Futsal App)](https://www.youtube.com/playlist?list=PLIan8aHxsPj2NeWJew3o86bSptVPXOppa) - Wahidev Academy
-
+* [Setup Redux Thunk Pada Project React](https://www.youtube.com/watch?v=frwlr7uh0io) - prawito hudoro
+* [Ayo Kita Bahas Middleware](https://www.youtube.com/watch?v=kVV17VI9qTU) - Parsinta
```

Comment:
- Using Youtube playlist where `frwlr7uh0io` belongs to.
- Adding the instructor @irsyadadl as well
- Recover blank line after list

```suggestion
* [Mari Kita Belajar Basic React JS](https://www.youtube.com/playlist?list=PLRKMmwY3-5MwXT8zMPbezhDnTM3cTA5cZ) - Irsyad A. Panjaitan, Parsinta

```

Reply:
Hey, thanks for adding me.

Reply:
Solved at https://github.com/EbookFoundation/free-programming-books/pull/6313/commits/5e861a4e6d350e39626173986741e614947ce56f